packages/popup/src/pages/ConfirmTransaction/index.js

```diff
@@ -401,11 +403,17 @@ function ConfirmTransaction() {
               pendingAuthReq={pendingAuthReq}
               isHwAccount={isHwAccount}
               setAuthStatus={setAuthStatus}
+              setSendError={setSendError}
               setIsAppOpen={setIsAppOpen}
             />
           )}
           {(isHwAccount || sendStatus === HW_TX_STATUS.REJECTED) && (
-            <HwTransactionResult status={sendStatus} isDapp={isDapp} />
+            <HwTransactionResult
+              isHwAccount={isHwAccount}
```

Comment:
the `isHwAccount ` is not used in the `HwTransactionResult` Component?

Reply:
done